src/components/TagModal.vue

```diff
@@ -163,6 +163,8 @@ export default {
 		},
 		async createTag(event) {
 			this.editing = true
+			if (this.showSaving) return
```

Comment:
It would be more elegant if the button is disabled while the previous async request is still happening. Then `createTag` can never be called while `this.showSaving` is true ;)

Reply:
Right, I added a commit disabling the input when needed, but leaving the `showSaving` condition in `createTag` for safety, or do you reckon that should be removed to simplify?

Reply:
Fair enough. We can do both!